src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/NuGetTelemetryProvider.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using NuGet.Common;
+
+namespace NuGet.VisualStudio.Telemetry
+{
+    [Export(typeof(INuGetTelemetryProvider))]
+    internal class NuGetTelemetryProvider : INuGetTelemetryProvider
+    {
+        public Task EmitEventAsync(TelemetryEvent telemetryEvent)
```

Comment:
nit: Probably doesn't need to return Task. It's also a private API, so there's no "maybe in the future" questions :)